@navikt/core/react/src/typography/ErrorMessage.tsx

```diff
@@ -51,7 +68,10 @@ export const ErrorMessage: OverridableComponent<
           "navds-label--small": size === "small",
         },
       )}
-    />
+    >
+      {icon && <ErrorMessageIcon />}
```

Comment:
Don't really see a need for `ErrorMessageIcon` having its own file. Think we can just inline it here 🤔